docs/ldap.md

```diff
@@ -29,11 +29,23 @@ authentication source:
     'example-ldap' => [
         'ldap:Ldap',
 
-        /* The hostname of the LDAP server. */
-        'hostname' => 'ldap.example.org',
+        /* The ldap-extension to use. Defaults to 'ext_ldap' */
+        'extension' => 'ext_ldap',
+
+        /* The connection string for the LDAP-server. You can add multiple by separating them with a space. */
+        'connection_string' => 'ldap.example.org',
+
+        /* Whether SSL/TLS should be used when contacting the LDAP server. Possible values are 'ssl', 'tls' or 'none' */
+        'encryption' => 'ssl',
```

Comment:
Would 'tls' be a more modern default?

Reply:
It doesn't say anything about actual TLS/SSL-versions..  They're actually different mechanisms for secure LDAP.
`ssl` in this case is a secure tunnel with LDAP traffic, usually on port  636.
`tls` is the STARTTLS variant where you would connect insecurely to port 389 where then TLS is negotiated as part of the LDAP-protocol. Call it 'stepup' if you will.

I'm not sure which one is the de facto standard nowadays, but I suspect it is `ssl`.. Active Directory domain controllers actively refuse connections to port 389 since last year.